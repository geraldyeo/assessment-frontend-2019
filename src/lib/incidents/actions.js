export const CREATE_ONE_INCIDENT = 'CREATE_ONE_INCIDENT';

export const createOneIncident = (payload) => ({ type: CREATE_ONE_INCIDENT, payload });