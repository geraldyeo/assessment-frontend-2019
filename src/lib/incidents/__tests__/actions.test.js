import {createOneIncident} from '../actions';

describe('Incident Actions', () => {
  test('createOneIncident should return an action object', () => {
    const payload = { key: 'bcg_1', title: 'First incident', assignee: 'Admin', status: true };
    expect(createOneIncident(payload)).toMatchSnapshot();
  });
});