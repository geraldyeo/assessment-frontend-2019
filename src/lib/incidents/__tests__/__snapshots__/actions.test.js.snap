// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Incident Actions createOneIncident should return an action object 1`] = `
Object {
  "payload": Object {
    "assignee": "Admin",
    "key": "bcg_1",
    "status": true,
    "title": "First incident",
  },
  "type": "CREATE_ONE_INCIDENT",
}
`;
